### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.13 esercizio 2 (2 dicembre)

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/calcolo-delle-probabilita#335] [Esercizi M-Z]
<img width="950" alt="image" src="https://github.com/user-attachments/assets/4e4dfd78-73cb-4ac8-b600-eaeaf278364d">

**Comment by alem1105** [comment]
<img width="861" alt="image" src="https://github.com/user-attachments/assets/eba64021-8c2f-481d-9a36-e3d50f57069d">
